docs/content/configuration/sumdb.md

```diff
@@ -5,41 +5,53 @@ weight: 2
 
-This is quite important. Say you are a company that is running an Athens instance, and you don't want the world to know about where your 
-repositories live. For example, say you have a private repo under `github.com/mycompany/secret-repo`. In order to ensure that the Go client 
-does not send a request to `https://sum.golang.org/lookup/github.com/mycompany/secret-repo@v1.0.0` and therefore leaking your private import path to the public, you need to ensure that you tell Go to skip particular import paths as such: 
+Before you begin, you'll need to tell your Go toolchain to not look for checksums on particular import paths:
 
 ```
-GONOSUMDB=github.com/mycompany/* go build
+$ export GONOSUMDB=github.com/mycompany/*
 ```
 
-This will make sure that Go does not send any requests to the Checksum DB for your private import paths. 
-However, how can you ensure that all of your employees are building private code with the right configuration? 
+After this, you can run `go get github.com/mycompany/secret-repo` without having your Go tool making a request to `sum.golang.org`.
+
+Athens, in this case can help ensure that all private code flowing through it never goes to the Checksum DB. So as long as your employees are using Athens, then they will get a helpful reminder to ensure Their GONOSUMDB is rightly configured.
 
-Athens, in this case can help ensure that all private code flowing through it never goes to the Checksum DB. So as long as your employees are using Athens, then they will get a helpful reminder to ensure Their GONOSUMDB is rightly configured. 
+As the Athens company maintainer, you can run Athens with the following configuration in your `config.toml` file.
 
-As the Athens company maintainer, you can run Athens with the following configuration: 
+```toml
+NoSumPatterns = ["github.com/mycompany/*"]
+```
+
+Or, you can set the `ATHENS_GONOSUM_PATTERNS` environment variable:
+
+```console
+$ export ATHENS_GONOSUM_PATTERNS="github.com/mycompany/*"
+```
 
-`NoSumPatterns = ["github.com/mycompany/*] # or comma separted env var: ATHENS_GONOSUM_PATTERNS`
+>The environment variable is a comma-separated list, so if you want to add more paths, your string should look like this: `"github.com/mycompany/*,github.com/myothercompany/*`
 
-This will ensure that when Go sends a request to `<athens-url/sumdb/sum.golang.org/github.com/mycompany/secret-repo@v1.0.0>`, Athens will return a 403 and failing the build ensuring that the client knows something is not configured correctly and also never leaking those import paths
+These steps will ensure that your Go toolchain sends a request to `<athens-url>/sumdb/sum.golang.org/github.com/mycompany/secret-repo@v1.0.0>`. When it does, Athens will return a 403 response code. All users who have not set up their `GONOSUMDB` environment variable will not be able to run `go get` for any of your private modules, and your private module names will not be leaked.
```

Comment:
I think also this line could be altered for clarity, the 403 would be for users who have not set their environment right?
Does s/response code. All users who have/response code for users who have/ or similar make more sense?